docs/topics/forms/modelforms.txt

```diff
@@ -753,8 +752,8 @@ Model formsets
 .. class:: models.BaseModelFormSet
 
 Like :doc:`regular formsets </topics/forms/formsets>`, Django provides a couple
-of enhanced formset classes that make it easy to work with Django models. Let's
-reuse the ``Author`` model from above::
+of enhanced formset classes that work well with Django models. Let's reuse the
```

Comment:
"work well" is a bit of a value judgment. I think "to make working with Django models more convenient." carries the intended meaning better.